src/main/java/liv/container/TaskList.java

```diff
@@ -0,0 +1,69 @@
+package liv.container;
+
+import java.util.ArrayList;
+
+import liv.task.Task;
+
+/**
+ * A class that stores the tasks from users.
+ */
+public class TaskList {
```

Comment:
I liked that you followed the naming convention of PascalCases for Java class names